models/Staticroute.php

```diff
@@ -451,32 +450,26 @@ public function assemble(array $urlOptions = [], $reset = false, $encode = true)
             }
         }
 
-        if ($reset) {
-            $requestParameters = [];
-        } else {
-            $requestParameters = \Pimcore::getContainer()->get('pimcore.routing.router.request_context')->getParameters();
+        $requestParameters = [];
 
-            // merge route params from static routes here
-            $request = \Pimcore::getContainer()->get('request_stack')->getCurrentRequest();
-            if (null !== $request && $request->attributes->get('_route_params')) {
-                $requestParameters = array_merge($requestParameters, $request->attributes->get('_route_params'));
-            }
+        // merge route params from static routes here
```

Comment:
@dvesh3 This entire code block can go I guess, it was used in the `else` clause and therefore only used if `$reset = false`.

Reply:
you are right...kicked it

Reply:
Thanks 😊